nlmod/gwf/wells.py

```diff
@@ -199,11 +200,26 @@ def maw_from_df(
     perioddata = []
 
     iw = 0
-    for index, irow in df.iterrows():
+    for index, irow in tqdm(df.iterrows(), total=len(df), desc="Adding MAW wells"):
         wlayers = np.where(multipliers[index] > 0)[0]
 
         # [wellno, radius, bottom, strt, condeqn, ngwfnodes]
-        pakdata = [iw, irow[rw], irow[botm], strt, condeqn, len(wlayers)]
+        if strt is None:
+            if isinstance(irow["cellid"], (np.integer, int)):
+                wstrt = (
+                    gwf.dis.top[irow["cellid"]]
+                    if ds is None
+                    else ds.top.values[irow["cellid"]]
+                )
+            else:
+                wstrt = (
+                    gwf.dis.top[irow["cellid"][0], irow["cellid"][1]]
+                    if ds is None
+                    else ds.top.values[irow["cellid"][0], irow["cellid"][1]]
+                )
```

Comment:
Can you make regular if-statements from these lines? That would be easier to read than these one-line if statements that rae sprread over multuiple lines.

Reply:
Can you change the docstring of strt, and inform the user with what value strt is replaced when strt is None?